packages/d3fc/package.json

```diff
@@ -29,7 +29,8 @@
     "@d3fc/d3fc-brush": "^3.0.3",
     "@d3fc/d3fc-chart": "^5.0.6",
     "@d3fc/d3fc-data-join": "^6.0.3",
-    "@d3fc/d3fc-discontinuous-scale": "^4.0.2",
+    "@d3fc/d3fc-discontinuous-scale": "^4.0.1",
+    "@d3fc/d3fc-domain-zoom": "^1.0.0",
```

Comment:
How about simply `d3fc-zoom`? It's unlikely we'll create a different zoom implementation.